README.md

```diff
@@ -1,230 +1,279 @@
 # hello-nodejs-express
 
-This quickstart consists of a basic hasura project with a simple  nodejs express app running on it. Once this project is deployed on to a hasura cluster, you will have the nodejs app running at https://api.cluster-name.hasura-app.io
+This quickstart consists of a basic [hasura project](https://docs.hasura.io/0.15/manual/project/index.html#hasura-project) with a simple nodejs express app running on it. Once this project is deployed, you will have the nodejs app running on your [cluster](https://docs.hasura.io/0.15/manual/getting-started/index.html#concept-2-a-hasura-cluster).
 
-This is the right place to start if you are planning to build or want to learn to build a react app with hasura.
+Follow along below to learn about how this quickstart works and to know how to tweak it for your needs.
 
-## Sections
+### Prerequisites
 
-* [Introduction](#introduction)
-* [Quickstart](#quickstart)
-* [Data API](#data-apis)
-* [Auth API](#auth-apis)
-* [Filestore API](#filestore-apis)
-* [Custom service](#custom-service)
-* [Migrate from an existing app](#migrate-from-an-existing-app)
-* [Local development](#local-development)
-* [Project Structure](#project-structure)
+* Ensure that you have the [hasura cli](https://docs.hasura.io/0.15/manual/install-hasura-cli.html) tool installed on your system.
 
-## Introduction
+```sh
+$ hasura version
+```
+
+Once you have installed the hasura cli tool, login to your Hasura account
+
+```sh
+$ # Login if you haven't already
+$ hasura login
+```
 
-This quickstart project comes with the following by default:
-1. A basic hasura project
-2. Two tables `article` and `author` with some dummy data
-3. A basic nodejs-express app which runs on the `api` subdomain.
+* You should have [Node.js](https://nodejs.org/en/) installed on your system, you can check this by:
 
-## Quickstart
+```sh
```

Comment:
Can't we compress this entire section to just the list of tools that need to be installed.